README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
+https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/change_feed/README.md.
+
 Populate your database:
 
     $ ./manage.py sync_couch_views
     $ ./manage.py migrate --noinput
-    $ ./manage.py collectstatic --noinput
```

Comment:
:+1: this is a huge misnomer for anyone getting started with django. i remember trying to edit the files in the staticfiles directory...